### DISCUSSION TITLE
/.pyenv/versions/3.7.0/lib/python3.7/site-packages/flask_caching/__init__.py:192: UserWarning: Flask-Caching: CACHE_TYPE is set to null, caching is effectively disabled.   "Flask-Caching: CACHE_TYPE is set to null, " Illegal instruction

### DISCUSSION THREAD

**Discussion by twointum** [apache/superset#12883] [Q&A / Help]
Installation is getting hung right on superset fab create-admin. Any ideas what is going on here? Thank you!

**Comment by nytai** [comment]
This is just a warning, it's unrelated. I think the command you're looking for is `flask fab create-admin`

**Reply by twointum** [reply]
Resulted in this:
`Error: Could not locate a Flask application. You did not provide the "FLASK_APP" environment variable, and a "wsgi.py" or "app.py" module was not found in the current directory.
`

**Reply by twointum** [reply]
Ran it again after running this line: export FLASK_APP=superset 
I get through the admin configuration prompts and get this:
`
Password:
Repeat for confirmation:
logging was configured successfully
INFO:superset.utils.logging_configurator:logging was configured successfully
/home/somakun/.pyenv/versions/3.7.0/lib/python3.7/site-packages/flask_caching/__init__.py:192: UserWarning: Flask-Caching: CACHE_TYPE is set to null, caching is effectively disabled.
  "Flask-Caching: CACHE_TYPE is set to null, "
Illegal instruction
`

**Reply by nytai** [reply]
ah I see you're running this via `pip install apache-superset`, https://superset.apache.org/docs/installation/installing-superset-from-scratch#installing-and-initializing-superset. `superset fab create-admin` should work then. You managed to run `superset db upgrade` successfully?

**Reply by twointum** [reply]
It didn't do anything but create the same error:

`
```
@debian:~$ superset fab create-admin
logging was configured successfully
INFO:superset.utils.logging_configurator:logging was configured successfully
/home/somakun/.pyenv/versions/3.7.0/lib/python3.7/site-packages/flask_caching/__init__.py:192: UserWarning: Flask-Caching: CACHE_TYPE is set to null, caching is effectively disabled.
  "Flask-Caching: CACHE_TYPE is set to null, "
Illegal instruction

```
`

**Reply by nytai** [reply]
But `superset db upgrade` worked?

**Reply by twointum** [reply]
neither work, same error string for both.

**Reply by nytai** [reply]
That isn't an error, it's just a warning. Ok, so those commands are just hanging for some reason. You might have better luck with docker-compose https://superset.apache.org/docs/installation/installing-superset-using-docker-compose

**Reply by twointum** [reply]
Yeah I tried that, It's not working either because I have a postgres instance already running on my server :)

Here is my existing question on that one : https://github.com/apache/superset/discussions/12876
